pkg/scheduler/framework/session.go

```diff
@@ -320,6 +322,29 @@ func (ssn *Session) Backoff(job *api.JobInfo, event arbcorev1.Event, reason stri
 	return nil
 }
 
+// UpdatePodCondition will update pod status of all pending tasks of a job
+func (ssn *Session) UpdatePodCondition(job *api.JobInfo) error {
```

Comment:
rename to `TaskUnschedulable`

Reply:
It should be `TaskUnschedulable(task *api.TaskInfo)`, and it's called if pod failed to be scheduled; in backoff, we should update task unschedulable for allocated tasks.

BTW, all pod related info should be handled in cache; session will use cache's interface to update pods.

Reply:
I will make the change.